src/transforms/BindAttributeTransform.js

```diff
@@ -26,7 +26,7 @@ module.exports = class BindAttributeTransform {
         const jsxName = PathUtils.getJSXElementName(path);
 
         // Check for certain special attributes on an input element (value, checked, indeterminate)
-        if (jsxName === 'input') {
+        if (jsxName === 'input' || jsxName === 'textarea') {
```

Comment:
Shouldn't this be on line 43 instead (`jsxName === 'select' || || jsxName === 'textarea'`)? You don't need to look at the type field for textarea